walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -39,46 +41,90 @@
                 }
             }
         }
+        List<TimeRange> availableTimes = getAvailableSlots(bookedTimes, request.getDuration());
```

Comment:
just for better readability I would rename availableTimes and availableOptionalTimes to requiredTimes and optionalTimes respectively.

Reply:
Agreed. Will implement

Reply:
Resolved